src/Integration.Vsix/CFamily/CMake/CompilationDatabaseRequest.cs

```diff
@@ -33,10 +34,19 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily.CMake
     internal class CompilationDatabaseRequest : IRequest
```

Comment:
I haven't changed the tests yet, so they're using the inner class as an integration.
I will be done with the vcx request's tests and then adopt CompilationDatabaseRequestTests.